src/components/DocsLink/DocsLink.tsx

```diff
@@ -4,10 +4,10 @@ import { Icon, useStyles2 } from '@grafana/ui';
 import { css, cx } from '@emotion/css';
 
 const docs = {
-  probes: `https://grafana.com/docs/grafana-cloud/monitor-public-endpoints/#probes`,
-  publicProbes: `https://grafana.com/docs/grafana-cloud/monitor-public-endpoints/probes/`,
-  privateProbes: `https://grafana.com/docs/grafana-cloud/monitor-public-endpoints/private-probes/`,
-  addPrivateProbe: `https://grafana.com/docs/grafana-cloud/monitor-public-endpoints/private-probes/#add-a-new-probe-in-your-grafana-instance`,
+  probes: `https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/#probes`,
```

Comment:
I suggest replacing the `synthetic-monitoring/#probes`link with the link to the public probes page.

Reply:
This particular URL should be changed because the content of this page has been migrated to a new [Introduction](https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/introduction) docs page.

Two choices:

1) Change it to https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/introduction/#probes

2) Change it to https://grafana.com/docs/grafana-cloud/monitor-public-endpoints/probes/

I lean towards option 2 because I think it is less error prone. 

- When migrating a section (heading), you cannot create a redirect. 
- A page often refer to an entire topic with more additional information. If the page for the concept/topic is relocated, we'll setup a redirect. 

Anyway, it is minor issue. lmk which one to set or feel free to change it.  Thanks!

Reply:
I don't have a strong preference, whichever you think works better is good!